microsoft/validation/yt8m_inference_json_formatter.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Microsoft All Rights Reserved.
+#
+# Active Learning Team
+# ==============================================================================
+
+r"""A simple helper script that formats Audioset Inference CSV files to a norma-
+lized consumable json to be used in model inference result comparisons. The scr-
+pt expects a directory of csv files processed my the Youtube-8m Frame Level mod-
+els and outputs a directory of respected json files converted by this script.
+Usage:
+  # Run a directory of csv files as the input.
+  # Specify a targeted output directory for the json files.
+
+  $ python yt8m_inference_json_formatter.py --csv_files /path/to/csv/files \
+                                    --json_files /path/to/targed/json/file/directory \
+
+  # Future code will include: Model Metrics, Threshold for Audioset Label Confidence
+"""
+from __future__ import print_function
+
+import os
+import csv
+import glob
+import json
+import pandas as pd
+import tensorflow as tf

```

Comment:
Remove

Reply:
Im using Pandas in line [52](https://github.com/evanlouie/fox-audio/blob/nate/audioDev/microsoft/validation/yt8m_inference_json_formatter.py#L51) and json in line [83](https://github.com/evanlouie/fox-audio/blob/nate/audioDev/microsoft/validation/yt8m_inference_json_formatter.py#L83).
Will remove the rest

Reply:
`pandas` and `json` are fine since you need them for your code. 
I was mentioning `tensorflow` as we don't really need it (if we migrate to `argsparser`)

Reply:
[Removed and switched to argsparser](https://github.com/evanlouie/fox-audio/blob/nate/audioDev/microsoft/validation/yt8m_inference_json_formatter.py#L23)